modules/virt-configuring-disk-sharing-lun.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-disk-sharing-lun_{context}"]
+= Configuring disk sharing by using LUN
+
+You can configure a LUN-backed virtual machine disk to be shared among multiple virtual machines by enabling SCSI persistent reservation. Enabling the shared option allows you to use advanced SCSI commands, such as those required for a Windows Failover Clustering implementation, against the underlying storage. Any disk to be shared must be in block mode.
+
+A disk of type `LUN` exposes the volume as a LUN device to the VM. This allows the VM to execute arbitrary iSCSI command passthrough on the disk.
+
+You reserve a LUN through the SCSI Persistent Reserve options to protect data on the VM from outside access. To enable the reservation, you configure the feature gate option. You then activate the `PersistentReservation` option on the LUN disk to  issue SCSI device-specific input and output controls (IOCTLs) that the VM requires.
+
+.Prerequisites
+
+* The volume access mode must be `ReadWriteMany` (RWX) if the VMs that are sharing disks are running on different nodes.
++
+If the VMs that are sharing disks are running on the same node, `ReadWriteOnce` (RWO) volume access mode is sufficient.
+* The storage provider must support a Container Storage Interface (CSI) driver that uses the SCSI protocol.
+* If you are a cluster admin and intend to configure disk sharing by using LUN, you must enable the cluster's feature gate on the Hyperconverged Cluster Operator Custom Resource.
+
+.Procedure
+
+. Edit or create the `VirtualMachine` manifest for your VM to set the required values, as shown in the following example:
++
+[source,yaml]
+====
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: testvmi-lun
+spec:
+  template:
+    spec:
+      domain:
+        resources:
+          requests:
+            memory: 64M
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: rootdisk
+            disk1: disk_one
+            shareable: true <1>
+          - disk:
+              bus: virtio
+            name: cloudinitdisk
+            lun: lun1 <2>
+            bus: scsi
+            reservation: true <3>
```

Comment:
The indentation here doesn't seem to be correct. Please check the example in the KubeVirt upstream documentation: https://kubevirt.io/user-guide/virtual_machines/disks_and_volumes/#lun